packages/mdx/src/index.ts

```diff
@@ -156,12 +158,29 @@ export const Themed: ThemedDiv & ThemedComponentsDict = styled('div')(
   themed('div')
 ) as ThemedDiv & ThemedComponentsDict
 
+// to be deprecated
```

Comment:
Okay, I think this is **already deprecated**, and deprecated means "to be removed in future releases".

We can use a JSDoc comment to mark that and point to the new name.
https://jsdoc.app/tags-deprecated.html

```suggestion
/**
 * @deprecated since 0.6.0.
 * 
 * `Styled` was renamed to `Themed` to avoid confusion with styled components.
 */
```

This is how it shows up in user's editors.

![image](https://user-images.githubusercontent.com/15332326/105643641-4ec90d80-5e91-11eb-8ae5-a1fc8ab703c8.png)